src/background/raven.js

```diff
@@ -69,7 +69,7 @@ export function init(config) {
 /**
  * Report an error to Sentry.
  *
- * @param {Error} error - An error object describing what went wrong
+ * @param {Object} error - An error object describing what went wrong
```

Comment:
The argument passed to this function was not always an `Error` and the code checked for this. This change just updates the documentation to reflect that.

Reply:
What about `Error|object`? The net effect is the same but it is more informative to the developer.

It is recommended to use `object` instead of `Object`: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html

Reply:
I don't actually know if it save to assume that `error` will be an object as opposed to a primitive in this context. This is a typical case of enabling TypeScript turning up existing uncertainty / ambiguity in the code. I'll leave it as `Object`/`any` for now to make it clear that it can't be assumed.